libs/components/forms/src/lib/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -1244,31 +1281,41 @@ describe('Checkbox component', () => {
     it('should set span class based on checkbox type input', () => {
```

Comment:
I know this wasn't you.... but could we potentially clean up this description a bit. Now that there are multiple `span` elements - it isn't very descriptive. Could we reword this to something like `should set the switch control's class based on checkbox type input`?

Reply:
good catch!